dspace/bin/make-handle-config

```diff
@@ -51,6 +51,7 @@ echo "Writing simple Handle server configuration"
 
 # Read parameters from DSpace config
 dshostname=`$BINDIR/dspace dsprop --property dspace.hostname`
+dshostip=`host $dshostname | awk '/has address/ { print $4; exit }'`
```

Comment:
Thanks for noting that.

DSpace still depends on handle-6.2.jar.

There is no Windows version that I know of.  Probably what we should do is deprecate this script in 6.0 and drop it in 7.0.  It is fragile and the problem it solves is not a very big problem.  We might replace it with more rugged, cross-platform Java code if it is really necessary.